src/CattyTests/Bricks/WaitBrickTests.swift

```diff
@@ -64,7 +64,7 @@ final class WaitBrickTests: XCTestCase {
         waitBrick.script = script
         let executionTime = self.measureExecutionTime(instruction: waitBrick.instruction(), expectation: nil)
         XCTAssertEqual(executionTime, duration, accuracy: 0.5, "Wrong execution time")
-        XCTAssertEqual(scheduler._activeTimers.count, 0)
+        XCTAssertEqual(scheduler.synchronizedTimerArray.count, 0)
     }
 
     func testSchedulerSetTimer_activeTimersCountIsOne() {
```

Comment:
Please remove the deprecated `_activeTimers` (i.e., `testSchedulerRegisterTimer`).

Reply:
Thank you, I've removed it!